### ISSUE TITLE
Change `age` field to date of birth

### ISSUE THREAD

**Issue by NicMcPhee** [UMM-CSci-3601/3601-iteration-template#1459]
Entering and storing age is kinda silly because that obviously won't _age_ well. We should instead be storing date of birth and computing age from that. Thanks to esitsu@Twitch for noting this.

**Comment by kklamberty** [comment]
I agree that it doesn't age well, but if we want to test for a certain age (which seems like a thing that is realistically likely to be a thing someone might want to do), the tests will not age well if the age is based on an actual date. Do we want to do something fancy in our seed and make the birthdates all be summer birthdates (so as not to cause trouble in either semester of the academic year), but then calculate the year for the birthdate to use in the seed to set those dates?
